certbot-dns-cloudflare/docs/conf.py

```diff
@@ -111,7 +111,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
(Here and for all other similar lines in the other `conf.py` files): I think we can remove this line if it not used.

Reply:
In my experience its fairly standard to leave these parameters in the config file and comment them out when not used. I have no issue removing them outside of that; just want to double check.